bucket_86/aucatctl/specification

```diff
@@ -0,0 +1,37 @@
+DEF[PORTVERSION]=	0.1
+# ----------------------------------------------------------------------------
+
+NAMEBASE=		aucatctl
+VERSION=		${PORTVERSION}
+KEYWORDS=		audio
+VARIANTS=		standard
+SDESC[standard]=	Control sndiod and/or aucat volumes
+HOMEPAGE=		http://www.sndio.org/
+CONTACT=		nobody
+
+DOWNLOAD_GROUPS=	main
+SITES[main]=		http://www.sndio.org/
+DISTFILE[1]=		aucatctl-${PORTVERSION}.tar.gz:main
+
+SPKGS[standard]=	single
+
+OPTIONS_AVAILABLE=	none
+OPTIONS_STANDARD=	none
+
+LICENSE=		ISCL:single
+LICENSE_FILE=		ISCL:{{WRKSRC}}/LICENSE
```

Comment:
set this to "ISCL:{{WRKDIR}}/LICENSE" to confirm to defacto standard (WRKDIR, not WRKSRC)

Reply:
ok

Reply:
done